### ISSUE TITLE
0.10.0 breaking change: `HasSystemDecorations="False"` causes window to be full screen on MacOS

### ISSUE THREAD

**Issue by derekantrican** [AvaloniaUI/Avalonia#5348]
Window:

```xml
<Window xmlns="https://github.com/avaloniaui"
        xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
        xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
        xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
        xmlns:gif="clr-namespace:AvaloniaGif;assembly=AvaloniaGif"
        mc:Ignorable="d" Width="350" Height="200"
        x:Class="Bulk_Edit_Calendar_Events_WPF.Windows.Loading"
        WindowStartupLocation="CenterScreen"
        TransparencyLevelHint="AcrylicBlur" Background="Transparent"
        HasSystemDecorations="False">
  <Grid>
    <Grid.ColumnDefinitions>
      <ColumnDefinition Width="*"/>
      <ColumnDefinition Width="Auto"/>
    </Grid.ColumnDefinitions>
    <TextBlock Text="Bulk&#x0a;Edit&#x0a;Calendar&#x0a;Events"
               Foreground="White"
               VerticalAlignment="Center"
               FontFamily="Britannic Bold" FontSize="35" Margin="5"/>
    <Image Grid.Column="1" HorizontalAlignment="Right"
           gif:GifImage.SourceStream="{Binding CalendarLoadingStream}"/>
  </Grid>
</Window>
```

Window looks as expected on Windows (Windows 10):
![](https://i.imgur.com/SAOCmhC.jpg)

But this is how it looks on MacOS (10.13.6 High Sierra):
![](https://i.imgur.com/nxa2ol9.png)

**Comment by derekantrican** [comment]
`SystemDecorations="None"` seems to have the same issue

**Comment by derekantrican** [comment]
@danwalmsley (not sure exactly who to tag, but want to let the Avalonia team know in case this truly is a significant break and not just the way that I'm doing things)

**Comment by sn4k3** [comment]
got same problem months ago: https://github.com/AvaloniaUI/Avalonia/issues/4879

**Comment by FrankenApps** [comment]
As a workaround you can set the windows' width and height from the CodeBehind like that:
```
public class MainWindow : Window {

  public MainWindow() {
        this.InitializeComponent();

        this.Height = 400;
        this.Width = 600;
   }

  private void InitializeComponent() {
            AvaloniaXamlLoader.Load(this);
   }
}
```

**Comment by derekantrican** [comment]
@FrankenApps Thanks, that definitely solves the sizing issue. Still will need to do some testing to figure out why `TransparencyLevelHint="AcrylicBlur"` isn't working for this window on OSX

**Comment by derekantrican** [comment]
@maxkatz6 I'll double-check

**Comment by derekantrican** [comment]
Yup seems to be fixed (I'm on 0.10.6). Thanks for the update!
